app/views/recipes/index.html.erb

```diff
@@ -7,15 +7,17 @@
   </div>
 <% end %>
 
-<% if @recipes.present? %>
-  <div id="recipes">
-    <% @recipes.each do |recipe| %>
-      <div class="recipe">
-        <h2><%= link_to recipe.title, recipe_path(recipe.id) %></h2>
-        <%= image_tag(recipe.image, alt: recipe.title, height: "100") if recipe.image.present? %>
-      </div>
-    <% end %>
-  </div>
-<% else %>
-  <p>No recipes found. Try a different search!</p>
+<% if params[:query].present? %>
+  <% if @recipes.present? %>
```

Comment:
do you need .present? here? Also, on the backend I added error handling for an empty reply from the API call, which we can encorporate. The JSOn response says something like: "No results match those key words"